pkg/collector/corechecks/gpu/gpu.go

```diff
@@ -62,10 +67,32 @@ func (m *Check) Configure(senderManager sender.SenderManager, _ uint64, config,
 		return fmt.Errorf("invalid gpu check config: %w", err)
 	}
 
+	// Initialize NVML collectors
+	m.nvmlLib = nvml.New()
```

Comment:
match the [logic](https://github.com/DataDog/datadog-agent/blob/888d4664522574433898e1ce7499226f44cbd6d8/cmd/system-probe/modules/gpu.go#L41-L45) from the probe

(including the config value for the path)

Reply:
Added config option